PMMain/Midas/MidasGUI/MidasGUI.vcxproj.filters

```diff
@@ -207,35 +265,37 @@
     <ClInclude Include="MidasCommon.h">
       <Filter>Header Files\Midas</Filter>
     </ClInclude>
-    <ClInclude Include="MouseCtrl.h">
-      <Filter>Header Files\Midas</Filter>
-    </ClInclude>
-    <ClInclude Include="MyoCommon.h">
-      <Filter>Header Files\Midas</Filter>
-    </ClInclude>
-    <ClInclude Include="MyoDevice.h">
-      <Filter>Header Files\Midas</Filter>
-    </ClInclude>
-    <ClInclude Include="MyoTranslationFilter.h">
-      <Filter>Header Files\Midas</Filter>
-    </ClInclude>
-    <ClInclude Include="SharedCommandData.h">
-      <Filter>Header Files\Midas</Filter>
-    </ClInclude>
-    <ClInclude Include="WearableDevice.h">
-      <Filter>Header Files\Midas</Filter>
-    </ClInclude>
-    <ClInclude Include="MidasMain.h">
-      <Filter>Header Files\Midas</Filter>
-    </ClInclude>
-    <ClInclude Include="SequenceImageManager.h">
+    <ClInclude Include="RingData.h">
       <Filter>Header Files</Filter>
     </ClInclude>
-    <ClInclude Include="SCDDigester.h">
-      <Filter>Header Files\Midas</Filter>
-    </ClInclude>
+  </ItemGroup>
+  <ItemGroup>
+    <CustomBuild Include="midasgui.h" />
+    <CustomBuild Include="midasgui.ui" />
+    <CustomBuild Include="DraggableWidget.h" />
+    <CustomBuild Include="GestureSignaller.h" />
+    <CustomBuild Include="InfoIndicator.h" />
+    <CustomBuild Include="MouseIndicator.h" />
+    <CustomBuild Include="SequenceDisplayer.h" />
+    <CustomBuild Include="MidasThread.h" />
+    <CustomBuild Include="midasgui.qrc" />
   </ItemGroup>
   <ItemGroup>
     <ResourceCompile Include="MidasGUI.rc" />
   </ItemGroup>
+  <ItemGroup>
+    <Text Include="keyboardData.txt" />
+  </ItemGroup>
+  <ItemGroup>
+    <Filter Include="Header Files">
+      <UniqueIdentifier>{93995380-89BD-4b04-88EB-625FBE52EBFB}</UniqueIdentifier>
+      <Extensions>h</Extensions>
+      <ParseFiles>true</ParseFiles>
+    </Filter>
+    <Filter Include="Source Files">
+      <UniqueIdentifier>{4FC737F1-C7A5-4376-A066-2A32D752A2FF}</UniqueIdentifier>
+      <Extensions>cpp;cxx;c;def</Extensions>
+      <ParseFiles>true</ParseFiles>
+    </Filter>
```

Comment:
Any idea why this file got changed so drastically? You only added 2 files. wouldnt have expected this.